skaffold.yaml

```diff
@@ -189,3 +199,33 @@ deploy:
         helm: {}
       valuesFiles:
       - example/gardener-local/gardenlet/values.yaml
+profiles:
+- name: kind2
+  patches:
+  # remove the check whether the local seed is ready because the garden cluster is not targeted for the kind2 profile
+  - op: remove
+    path: /deploy/helm/hooks/after/0
```

Comment:
Should we instead add a similar hook for waiting for the the `local2` seed to become ready? Maybe above script can be reused somehow?

Reply:
I could try to manually specify the kubeconfig to that of the first cluster inside the hook script.